.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+# E501 line too long
+# W503 line break before binary operator
+ignore = E501,W503
```

Comment:
We need to add `max-line-length = 88` and ignore `E203` according to black + flake8 compatibility docs: https://black.readthedocs.io/en/stable/the_black_code_style.html

Reply:
Replaced E501 ignoring by max line length 88 in 5ea5b77.

Ignored E203 in 7d30fb3 (although we don't need to ignore that rule yet as we have no problematic cases).